Atomic/pull.py

```diff
@@ -18,20 +19,27 @@ def cli(subparser):
                        help=_("Specify the storage. Default is currently '%s'.  You can"
                               " change the default by editing /etc/atomic.conf and changing"
                               " the 'default_storage' field." % backend))
+    pullp.add_argument("-t", "--type", dest="reg_type", default=None,
+                       help=_("Push to an alternative registry type."))
     pullp.add_argument("image", help=_("image id"))
 
 
 class Pull(Atomic):
     def pull_docker_image(self):
-        _, _, tag = decompose(self.args.image)
+        registry, _, tag = decompose(self.args.image)
+        insecure = True if is_insecure_registry(self.d.info()['RegistryConfig'], remove_port(registry)) else False
         # If no tag is given, we assume "latest"
         tag = tag if tag != "" else "latest"
-        fq_name = skopeo_inspect("docker://{}".format(self.args.image))['Name']
+        if self.args.reg_type == "atomic":
+            pull_uri = 'atomic:'
```

Comment:
@rhatdan we stripped the double slash long time ago when designing system containers IIRC, any reason we want it again? @mtrmac do you remember by any chance?